src/ui/server/controllers/insurance/export-contract/about-goods-or-services/index.ts

```diff
@@ -90,7 +89,7 @@ export const get = async (req: Request, res: Response) => {
 
     return res.render(TEMPLATE, {
       ...insuranceCorePageVariables({
-        PAGE_CONTENT_STRINGS: PAGES.INSURANCE.POLICY.ABOUT_GOODS_OR_SERVICES,
+        PAGE_CONTENT_STRINGS: PAGES.INSURANCE.EXPORT_CONTRACT.ABOUT_GOODS_OR_SERVICES,
```

Comment:
since we have changed this in so many places on this page, const on top for this?

Reply:
Good shout, created a new `const`.